### ISSUE TITLE
port to VCV rack

### ISSUE THREAD

**Issue by gbrandt1** [timchurches/Mutated-Mutables#6]
hi tim, has anybody contacted you about porting this firmware to VCV Rack?

**Comment by timchurches** [comment]
No, no-one.

**Comment by gbrandt1** [comment]
would you be opposed to the idea? i've recently completed a full port of Braids (essentially adding the missing options from the "official" Audible Instruments port) and would be interested to do it

**Comment by timchurches** [comment]
I don’t mind, although not sure why you would want to do it, given you can achieve most of what it does with external modulation, and in a virtual rack, there’s no scarcity of modulation sources. I think only the meta-sequencer feature would be worth adding to the existing virtual Braids.

**Comment by gbrandt1** [comment]
that's true to an extend, and if i did it, much of the functionality would be exposed on an extra "breakout" extension of the virtual panel instead of through the menus - which could then be just an external sequencer + lfos as well, in principle. but ok thanks, now i know it's not a waste of time to continue thinking about it, and i do like integrated things.
